cypress/e2e/saved.cy.js

```diff
@@ -1,18 +1,20 @@
 
 beforeEach(() => {
-  cy.intercept("GET", "https://meowfacts.herokuapp.com/?count=100", {
+  cy.intercept("GET", "https://meowfacts.herokuapp.com/?count=50", {
     statusCode:200,
     fixture:"facts"
-  })
+  }).as('fact')
   cy.intercept("GET", "https://cataas.com/cat/gif",{
     statusCode:200,
     fixture:"response.gif"
-  })
+  }).as('gif')
   cy.visit("http://localhost:3000/")
 })
 
 
 it('should allow the user to save a fact and delete the fact on the saved fact page', () => {
+  cy.wait('@fact')
+  cy.wait('@gif')
```

Comment:
This test seems to be failing almost half the times I've run it in cypress, I wonder if its because you're not using .then((intercept) => {...}) like you did in the error testing file?